crossplane/apiextensions_test.go

```diff
@@ -453,13 +453,11 @@ func TestCompositeResource(t *testing.T) {
 		t.Fatalf("Create client: %v", err)
 	}
 
-	// TODO(negz): Use the other provider-nop from contrib once it's ready.
-	// https://github.com/crossplane-contrib/provider-nop
 	prv := &pkgv1.Provider{
 		ObjectMeta: metav1.ObjectMeta{Name: internal.SuiteName},
 		Spec: pkgv1.ProviderSpec{
 			PackageSpec: pkgv1.PackageSpec{
-				Package:                     "negz/provider-nop:v0.1.0",
+				Package:                     "crossplane/provider-nop:main",
```

Comment:
Should we cut a release of provider-nop we can pin to? I can take care of doing that, since I believe it will require adding some secrets to the provider-nop GitHub repo.

Reply:
That will be great. Thanks!